tests/test_flow/test_logging.py

```diff
@@ -43,40 +42,61 @@ def x_plus_two(x_plus_one):
 
     flow = builder.build()
     assert flow.get("x_plus_one") == 2
-
     log_checker.expect(
         "Accessed   x(x=1) from definition",
         "Computing  x_plus_one(x=1) ...",
         "Computed   x_plus_one(x=1)",
     )
 
     assert flow.get("x_plus_two") == 3
-    log_checker.expect(
-        "Accessed   x_plus_one(x=1) from in-memory cache",
-        "Computing  x_plus_two(x=1) ...",
-        "Computed   x_plus_two(x=1)",
-    )
+
+    if parallel_processing_enabled:
+        # This is different from serial processing because we load entites
```

Comment:
s/entites/entities/

Reply:
Also, this doesn't really explain why they're different, it just restates what the assertion is saying.